### ISSUE TITLE
Add MNMG instructions to the Sagemaker page

### ISSUE THREAD

**Issue by jacobtomlinson** [rapidsai/deployment#112]
The Sagemaker page currently talks thoguh launching a notebook instance with GPUs and then running an example notebook.

It would be great if the page went into more details about how workloads and burst beyond the notebook instance to get more resource.

The example notebook uses an `Estimator` which launches a workload onto the Sagemaker batch infrastructure. It would be great to writer a little about how that process works:
- Write a script
- Build that into a docker image with RAPIDS and sagemaker tools
- Push to ECR
- Launch a job with that image

Another useful section would be about alternatives like launching a Kubernetes cluster and connection into it. We wouldn't need t  go into detail about how to do that, just link to the EKS page.

**Comment by skirui-source** [comment]
Hi @jacobtomlinson I think what you are requesting here is essentially what this[ blog ](https://medium.com/rapids-ai/running-rapids-experiments-at-scale-using-amazon-sagemaker-d516420f165b)covers in-depth? Wondering in what format did you want this documentation?

**Comment by skirui-source** [comment]
Other references:https://aws.amazon.com/blogs/machine-learning/rapids-and-amazon-sagemaker-scale-up-and-scale-out-to-tackle-ml-challenges/ and https://sagemaker-examples.readthedocs.io/en/latest/hyperparameter_tuning/rapids_bring_your_own/rapids_sagemaker_hpo.html

**Comment by jacobtomlinson** [comment]
Yeah that blog post is a good foundation for this documentation. However that post is three years old now so we should check that everything works as expected, I assume things will need updating.

Currently [our SageMaker documentation page](https://github.com/rapidsai/deployment/blob/main/source/cloud/aws/sagemaker.md) says "launch a notebook instance, then follow the higgs notebook or read this blog post". I'd like to have more content on the page about how `Estimators` work and how to use them before we direct folks away to workflow examples.

The blog post walks you through running a specific workflow. The deployment instructions are kinda woven throughout. The docs page should be those deployment instructions extracted and more simplified. It should answer questions like:

- Q: How do I launch a SNMG/MNMNG training job on SageMaker? A: Use an `Estimator`.
- Q: How do I use an `Estimator`? A: Write a script to run your work and package it in a container image.
- Q: What are the requirements for the container image? A: It needs your Python environment with the `sagemaker` package.
- Etc.

I like to think about documentation in terms of the [Diataxis framework](https://diataxis.fr/). In that model, the Blog Post is a _tutorial_ and the notebook is a _how-to_. The deliverable for this PR is a documentation page that covers the _explaination_. We can link to the [SageMaker docs pages](https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html) for _reference_.
